javascript/javascript1/week1/ez-namey.js

```diff
@@ -0,0 +1,11 @@
+// Ez Namey (Startup name generator) Optional
+/* At a meetup you overhear a conversation between two developers.
+ It went something like this:
+  "Man i wish picking a startup name was easier!
+   You need to be creative, funny and interesting, its nearly impossible!" Another problem to solve, awesome! */
+
+const firstWords = ["Center", "Creative", "Data", 'Founder', "Alpha", "Infinite", "Geo", "Inside", "Simple", "Sphere"];
+const secondWords = ["Webnet", "Weborate", "Webaboon", "Websiter", "Webfore", "Weberq", "Webitio", "Webarnup", "Weblogy", "Weboratory"];
+const randomNumber = Math.floor(Math.random() * 10) + 0;
```

Comment:
```suggestion
const randomNumber =  Math.floor(Math.random() *firstWords.length)
```
 Try not to use constants here. If the array size changes then you would have to modify the code here as well